src/components/Header/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { bool, func } from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { schema } from '../../utils/prop-types';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    backgroundColor: theme.palette.grey[300],

```

Comment:
I worry that this won't sit well for themes that don't work well with the gray background or that have `theme.palette.text.primary` set to something that would make it hard to read the text. Any idea what we should do here?

Reply:
We could use `theme.palette.getContrastText(theme.palette.text.primary)` so that the background will automatically use a contrast color of the text color. This would let users only have to customize the `theme.palette.text.primary` value when needed and the background color will be automated to an appropriate color (mostly light or dark).

| default theme | dark theme |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/29671309/74317950-a2953080-4dbf-11ea-80a1-47839621a69b.png) | ![image](https://user-images.githubusercontent.com/29671309/74319247-f3a62400-4dc1-11ea-9c01-d2242bfd282f.png) |
| __yellow theme__ | __grey theme__ |
| ![image](https://user-images.githubusercontent.com/29671309/74319295-09b3e480-4dc2-11ea-9d91-e5e987aaec07.png) | ![image](https://user-images.githubusercontent.com/29671309/74319157-d3766500-4dc1-11ea-8ca5-580db6500fc5.png) |

I've applied the changes in f53e186

Reply:
We could also further expand on using the `getContrastText()` so that the `type` blocks in the left rows sit well with the themes as well. Currently, the `type` blocks are set to a grey color so it may be hard to read with certain themes.

**current**
![image](https://user-images.githubusercontent.com/29671309/74320879-bd1dd880-4dc4-11ea-8abf-50d86eab43fb.png)

**with change**
| default theme | dark theme | yellow theme | 
|:---:|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/29671309/74320968-e0488800-4dc4-11ea-8dfe-b928fe03ef5b.png) | ![image](https://user-images.githubusercontent.com/29671309/74320929-d0c93f00-4dc4-11ea-8a5a-335f7276aace.png) | ![image](https://user-images.githubusercontent.com/29671309/74321100-1554da80-4dc5-11ea-9949-2c72b75e2617.png) |

If this is something we should use and further expand on, I would like to open a separate issue to deal with all the colors within the schemaTable since we would also have to deal with the border colors and comment colors as well.

Reply:
Makes sense. Feel free to create an issue to deal with that.